src/compiler/operations.md

```diff
@@ -1,6 +1,8 @@
 # Operations
 
-Here is a list of recurring tasks to support the compiler team and help keep things moving forward. Ideally run through this list every week. If there are blockers or doubts, after having acquired the right context, don't hesitate to ping people around. Keep in mind that contributors are the best resource of the project and we want to be mindful of their time.
+"Operations" is a part of the Compiler Team that takes care of organizational work and in general help things moving forward. T-compiler ops lives on Zulip under [#t-compiler/ops](#).
+
+Here is a list of recurring tasks. Ideally run through this list every week. If there are blockers or doubts, after having acquired the right context, don't hesitate to ping people around. Keep in mind that contributors are the best resource of the project and we want to be mindful of their time.
```

Comment:
Discussion: In addition to

> don't hesitate to ping people around

We might also document an "escalation" process: an ops member might further:

1. Open a dedicated zulip thread under `#t-compiler` about such topic, and/or
2. Nominate the topic for the weekly triage meeting if the topic needs some kind of decision in a reasonably timely matter (I'm thinking of maybe a critical regression?).

The idea is not to have something keep sitting around and not be resolved or be forgotten.